force-app/main/default/classes/ResourceStatusTrackerCtrl.cls

```diff
@@ -76,4 +75,17 @@ public with sharing class ResourceStatusTrackerCtrl {
             throw new AuraHandledException(JSON.serialize(e.getTypeName() + ': ' + e.getMessage() + ' - ' + e.getStackTraceString()));
         }
     }
+
+    public class StatusReport {
+        @AuraEnabled
+        public String namespace {get;set;}
+        @AuraEnabled
+        public Status_Report__c  record {get; set;}
+
+        public StatusReport(Status_Report__c statusReport) {
+            String namespace = UTIL_Namespace.getNamespace();
+            this.namespace = String.isBlank(namespace)? namespace:namespace + '__';
+            this.record = statusReport;
+        }
+    }
```

Comment:
Curious to know why do we need this change and the changes to resourceStatusTrackerHelper.js